app/views/procedures/new.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= form_for @procedure, url: {action: "create"} do |f|%>
```

Comment:
I believe it needs to be form_for [@proposal, @experiment, @procedure] to work properly, I had to nest it this way for my experiment form partial.